app/src/main/java/lang_import/org/app/ReaderActivity.kt

```diff
@@ -17,13 +18,42 @@ import android.widget.Toast
 import kotlinx.android.synthetic.main.activity_reader.*
 
 class ReaderActivity : AppCompatActivity() {
-    val reader by lazy { FeedReader("https://habr.com/rss/all/", this) }
+    //val reader by lazy { FeedReader("https://habr.com/rss/all/", this) }
+    //draft BD
+    val informersMap:HashMap <String,String> = hashMapOf(
+            "HABR" to "https://habr.com/rss/all/",
+            "Yandex.science" to "https://news.yandex.ru/science.rss",
+            "mail.ru" to "https://news.mail.ru/rss/"
+    )
+    var informerURL=""
+    val reader by lazy { FeedReader(informerURL, this) }
```

Comment:
A bit strange: if `reader` once invoked, then it is never initialized again even if `informerURL` changed

Reply:
Yes it is. Now we can use new informer only after restart app. This code is only concept for load multiple informers. Need refactor reader or add some refresh functional. Or we can not use multiple informers

Reply:
Maybe just remove lazy initialization and implement the getter?

Reply:
Done :)